training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb

```diff
@@ -1002,7 +1002,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.13"
+   "version": "3.8.10"
```

Comment:
Intentional?

Reply:
+1

Reply:
Oh nope, this is just because I have python3.8 on my computer. Will update, thanks!